stustapay/core/service/config.py

```diff
@@ -9,20 +9,33 @@
 from stustapay.core.service.common.error import NotFound
 
 
+async def get_currency_identifier(*, conn: asyncpg.Connection) -> str:
+    return await conn.fetchval("select value from config where key = 'currency.identifier' limit 1")
+
+
 class ConfigService(DBService):
     def __init__(self, db_pool: asyncpg.Pool, config: Config, auth_service: AuthService):
         super().__init__(db_pool, config)
         self.auth_service = auth_service
 
+    @with_db_transaction
+    async def is_sumup_topup_enabled(self, *, conn: asyncpg.Connection) -> bool:
+        db_config_entry = await conn.fetchval("select value from config where key = 'sumup_topup.enabled'")
+        return self.cfg.customer_portal.sumup_config.enabled and db_config_entry == "true"
```

Comment:
why have this information stored in the database and in the config? seems a bit redundant to me.